alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/AlertingPlugin.kt

```diff
@@ -131,7 +132,8 @@ internal class AlertingPlugin : PainlessExtension, ActionPlugin, ScriptPlugin, P
         environment: Environment,
         nodeEnvironment: NodeEnvironment,
         namedWriteableRegistry: NamedWriteableRegistry,
-        indexNameExpressionResolver: IndexNameExpressionResolver
+        indexNameExpressionResolver: IndexNameExpressionResolver,
+        repositoriesServiceSupplier: Supplier<RepositoriesService>?
```

Comment:
Is this actually nullable or just added from the autocomplete?

Reply:
Autocomplete.

Reply:
look forward to seeing what will be changed here, if something need to be changed.  :smile:

Reply:
Ok, can we remove the ? at the end then as it isn't actually nullable

Reply:
Done.